src/devices/vdml_motors.cpp

```diff
@@ -205,19 +205,31 @@ motor_pid_full_s_t Motor::convert_pid_full(double kf, double kp, double ki, doub
 }
```

Comment:
Might as well depreciate these as well, since they're useless without the set Pids.